src/themes/vegav/main.scss

```diff
@@ -23,6 +27,12 @@ $card_padding: 20px;
   margin-top: 20px;
 }
 
+.logoName span {
+  background-color: white;
+  padding: 50px;
+  border-radius: 10px;
+}
```

Comment:
Bitte in core/ neue Variablen anlegen:
- $logo-name_background
- $logo-name_padding
- $logo-name_borderRadius